keyboards/mexsistor/ludmila/matrix.c

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
Theres a fair amount here to digest, but in the short term, I would suggest <https://docs.qmk.fm/#/custom_matrix?id=lite> here to avoid duplicating all of commons code and fixing cases like matrix_scan_kb not calling _user.